editor/inspector_dock.cpp

```diff
@@ -553,8 +553,8 @@ InspectorDock::InspectorDock(EditorNode *p_editor, EditorData &p_editor_data) {
 	inspector->set_v_size_flags(Control::SIZE_EXPAND_FILL);
 	inspector->set_use_doc_hints(true);
 	inspector->set_hide_script(false);
-	inspector->set_enable_capitalize_paths(bool(EDITOR_DEF("interface/editor/capitalize_properties", true)));
-	inspector->set_use_folding(!bool(EDITOR_DEF("interface/editor/disable_inspector_folding", false)));
+	inspector->set_enable_capitalize_paths(bool(EDITOR_GET("interface/inspector/capitalize_properties")));
+	inspector->set_use_folding(!bool(EDITOR_DEF("interface/inspector/disable_inspector_folding", false)));
```

Comment:
I *still* think the second `_inspector` is redundant... :)

Reply:
Yeah, maybe, I'll test it here and if it's not necessary I'll remove than.

Reply:
You'are referrgin to `interface/inspector/capitalize_properties` right?

Reply:
No, I'm just saying that "interface/**inspector**/disable_**inspector**_folding" should be "interface/**inspector**/disable_folding".

"disable_folding" is enough if we're already in the "inspector" category, no need to restate it.

Reply:
I understood now, I fixed disable_folding to update without need to restart godot too.